packages/account/src/providers/operations.graphql

```diff
@@ -15,17 +15,26 @@ fragment transactionStatusFragment on TransactionStatus {
       id
     }
     time
+    receipts {
+      ...receiptFragment
```

Comment:
Adding receipts here requires updating the subscription logic to account for events that span multiple batches of streamed data because receipts can be arbitrarily large and can exceed the size of a single batch. I already have the logic for that implemented for #1644 but that branch is really old. I'll create a PR onto `master` with only that logic.

Reply:
Sorry, but I lost context — what are you saying exactly?

Reply:
Sorry for the vagueness. After looking at the code again I realized I mixed up two things. I tagged you thinking that we'd need to push for the implementation of the [issue](https://github.com/FuelLabs/fuel-core/issues/1724) that's blocking #1644, but that's not the problem here. I'll close this discussion and write another comment detailing the issue.

Reply:
Perfect, thank you.

Reply:
https://github.com/FuelLabs/fuels-ts/pull/2021#discussion_r1560930082